src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -102,7 +102,7 @@
   "Calls clojure.jdbc/db-query-with-resultset after adding (jdbc/db)
    as the first argument. Note that this will hold the whole resultset in memory
    due to the default jdbc fetchsize of 0. If streaming is required, use
-   with-query-results-cursor."
```

Comment:
Well, "vectors of column" (values), but yeah row vector might be more sensible -- I'll fix it in the next rebase.